actioncable/lib/assets/javascripts/action_cable/subscriptions.coffee

```diff
@@ -76,3 +78,10 @@ class ActionCable.Subscriptions
   toJSON: ->
     history: @history
     identifiers: (subscription.identifier for subscription in @subscriptions)
+
+  dispatchEvent: (subscription, callbackName, args...) ->
+    unless subscription.constructor.name == "ConnectionMonitor"
+      channel = JSON.parse(subscription.identifier).channel
+      event = new Event "cable:#{channel}:#{callbackName}"
```

Comment:
I might be wrong, but I don't think using the `Event` constructor works in IE. Should probably do it the old fashioned way:
```coffee
event = document.createEvent("Events")
event.initEvent(eventName, true, false)
document.dispatchEvent(event)
```